okhttp-clients/src/main/java/com/palantir/remoting3/okhttp/RemotingOkHttpCall.java

```diff
@@ -236,14 +272,18 @@ public Void visit(QosException.Throttle exception) {
                 }
 
                 Duration backoff = exception.getRetryAfter().orElse(nonAdvertizedBackoff.get());
```

Comment:
difficult to grok what the interplay is between the Limiter and the backoffStrategy. they seem duplicative.

Reply:
and yet, they are not :)

Reply:
exactly